settings.py

```diff
@@ -36,3 +36,8 @@
 # number of FlowMod requests. Use 0 (zero) to disable BATCH mode, i.e. sends
 # everything at a glance
 BATCH_SIZE = 50
+
+# Default values for EVPL and EPL respectively. They are use when sb_priority
+# is not set in a request
+EVPL_SB_PRIORITY = 1000
+EPL_SB_PRIORITY = 500
```

Comment:
@jab1982 has suggested to use higher values to give more space between the two of them:
```suggestion
EVPL_SB_PRIORITY = 20000
EPL_SB_PRIORITY = 10000
```

Reply:
Got it.

Reply:
Hey @Alopalao I saw that you changed the values on 868aee2 but you put 1000 and 2000, instead of 10000 and 20000 (there is one missing zero). Can you please fix this?

Reply:
Sorry, changed